src/cephadm/cephadm

```diff
@@ -967,7 +974,7 @@ def check_ip_port(ctx, ip, port):
             s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         try:
             attempt_bind(ctx, s, ip, port)
-        except OSError as e:
+        except PortOccupiedError as e:
             raise Error(e)
```

Comment:
Does an error of type `Error` needs to be raised here or would any other error also work?

Reply:
lgtm as it is